src/schema/v2/artist/verifiedRepresentatives.ts

```diff
@@ -0,0 +1,51 @@
+import { GraphQLFieldConfig, GraphQLList } from "graphql"
+import { ResolverContext } from "types/graphql"
+import { PartnerType } from "schema/v2/partner/partner"
+
+const VerifiedRepresentatives: GraphQLFieldConfig<
+  { _id: string },
+  ResolverContext
+> = {
+  type: GraphQLList(PartnerType),
```

Comment:
```suggestion
  type: new GraphQLNonNull(new GraphQLList(newGraphQLNonNull(PartnerType))),
```

Reply:
I think you have one space missing here `newGraphQLNonNull` 
It should be `type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(PartnerType))),`

Reply:
Yep!